app/controllers/application_controller.rb

```diff
@@ -7,7 +7,7 @@ class ApplicationController < ActionController::Base
 
   include GDS::SSO::ControllerMethods
 
-  before_filter :require_signin_permission!
+  before_action :require_signin_permission!
```

Comment:
I think `before_action` is available in Rails 4 (and deprecated in 5.1). If possible, could this change be made in a separate commit before the Rails upgrade?